### PR TITLE
Meenu/trade page table

### PR DESCRIPTION
**Pull request by meenakshi-deriv**

### CODE REVIEW

File: libs/components/src/lib/table-cell/table-header-cell/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  width,
+}) => {
+  return (
+    <TableSymbolCell
+      className={className}
+      header={header}
+      color="coral"
+      width={width}
+    ></TableSymbolCell>
```
Comment (prince-deriv) [inline libs/components/src/lib/table-cell/table-header-cell/index.tsx:16]:
```suggestion
  return (
    <TableSymbolCell
      className={className}
      header={header}
      color="coral"
      width={width}
    />
```

File: libs/components/src/lib/table-cell/table-symbol-cell/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        sizeVariants[width],
+        'p-general-xs lg:px-general-md',
+        'border-b-xs border-b-solid-slate-75',
+        'flex flex-col',
+        'min-h-[72px]',
+        'items-start  justify-center',
+        className,
+      )}
+    >
+      <div className={qtMerge('gap-gap-md', 'flex', 'items-center')}>
```
Comment (prince-deriv) [inline libs/components/src/lib/table-cell/table-symbol-cell/index.tsx:45]: Let's not use `qtMerge` if there are no other variable classnames to merge. Keep it a string instead

File: libs/templates/src/lib/trade/table/trading-spec-table.tsx [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+    getPaginationRowModel: getPaginationRowModel(),
+  });
+  useEffect(() => {
+    table.setPageSize(15);
+  }, [table]);
+  useEffect(() => {
+    document.body.style.overflow = showBottomSheet ? 'hidden' : 'scroll';
+  }, [showBottomSheet]);
+  return (
+    <Section
```
Comment (mohsen-deriv) [inline libs/templates/src/lib/trade/table/trading-spec-table.tsx:69]: can you please break this down to smaller components? 
this is gonna be really hard to maintain
thanks thanks 🙏

File: libs/blocks/src/lib/hero/content-less/index.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ const ContentLess = ({
         className,
       )}
     >
-      <FluidContainer className="flex flex-col gap-gap-3xl">
+      <FluidContainer className="flex flex-col gap-gap-2xl">
```
Comment (mohsen-deriv) [inline libs/blocks/src/lib/hero/content-less/index.tsx:25]: are you sure about this change? 
please check it with hero / content-less block in figma 🙏

File: libs/components/src/lib/action-sheet/bottom-sheet/index.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      className={qtMerge(
+        'fixed inset-y-gap-xs left-50 z-10 flex h-full w-full flex-col justify-end bg-typography-default ',
+      )}
+    >
+      <div
+        className={qtMerge(
+          'fixed w-full rounded-t-lg bg-solid-slate-50 p-general-sm',
+        )}
+      >
+        <div className="flex flex-row items-center justify-between p-general-md">
```
Comment (mohsen-deriv) [inline libs/components/src/lib/action-sheet/bottom-sheet/index.tsx:22]: please add a conditional rendering on top of this like: 
```
{header || icon } && <div className="flex flex-row items-center justify-between p-general-md">
</div>
```

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: libs/templates/src/lib/trade/table/trading-spec-table.tsx (hunk 3 above)
Comment (meenakshi-deriv) [reply libs/templates/src/lib/trade/table/trading-spec-table.tsx:69]: hi @mohsen-deriv chips and pagination will be done from quill design. i have used buttons with conditions here, that's why it looks huge.

File: libs/components/src/lib/action-sheet/bottom-sheet/bottom-sheet.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        </div>
+        <div
+          className={qtMerge('flex flex-row items-center gap-gap-md py-gap-md')}
+        >
+          <StandaloneCircleRegularIcon fill="black" iconSize="md" />
+
+          <Text size="md">Trading conditions</Text>
+        </div>
+        <div
+          className={qtMerge('flex flex-row items-center gap-gap-md py-gap-md')}
```
Comment (prince-deriv) [inline libs/components/src/lib/action-sheet/bottom-sheet/bottom-sheet.stories.tsx:43]:
```suggestion
          className="flex flex-row items-center gap-gap-md py-gap-md"
```

If there are no variable classname to merge, let's use the normal class declaration instead.

File: libs/components/src/lib/action-sheet/bottom-sheet/index.tsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+export const BottomSheet = ({
+  heading,
+  icon,
+  children,
+  placeholdericon,
+}: BottomSheetProps) => {
+  return (
+    <div className="fixed inset-y-gap-xs left-50 z-10 flex h-full w-full flex-col justify-end bg-typography-default ">
+      <div className="fixed w-full rounded-t-lg bg-solid-slate-50 ">
```
Comment (Nuzhy-Deriv) [inline libs/components/src/lib/action-sheet/bottom-sheet/index.tsx:19]:
```suggestion
      <div className="fixed w-full rounded-t-lg bg-background-primary-base">
```

File: apps/deriv-hk/pages/index.tsx [hunk 8]
```diff
@@ -6,6 +6,11 @@ export function Index() {
   return (
     <>
       <Head>
+        <meta
+          name="viewport"
+          key="viewport"
+          content="width=device-width, initial-scale=1.0"
+        />
```
Comment (prince-deriv) [inline apps/deriv-hk/pages/index.tsx:13]: We need this in all pages yeah?

Comment (meenakshi-deriv) [reply apps/deriv-hk/pages/index.tsx:13]: needed in homepage, help center and trade page. should we customize this?

Comment (mohsen-deriv) [reply apps/deriv-hk/pages/index.tsx:13]: can you please move this to `_document.tsx`?

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**